src/app/components/MediaLoader/utils/buildSettings.ts

```diff
@@ -28,15 +29,24 @@ const buildSettings = ({
   translations,
   adsEnabled = false,
   showAdsBasedOnLocation = false,
+  embedded,
+  pathname,
 }: BuildConfigProps) => {
+  const embedUrl = pathname
```

Comment:
Not sure whether this ought to be conditional on pathname. I added it for unit tests, and it would prevent an error in production if the `pathname` value was missing.

Reply:
Think its safe enough to check, does no harm really even if `pathname` should technically always be there.